README.md

```diff
@@ -6,13 +10,15 @@ Automated code review in Python done with Prospector
 
 Right now, Feito only supports GitHub.
 
-### 1) Requirements
+### Installation
+
+`$ pip install feito`
 
-An account in GitHub must have its OAuth Token acquired. Follow this [link](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token). When the Scopes section arrives, the `admin:repo_hook` and `repo` checkboxes must be selected. Copy the Token (really, copy it, you won't be able visualize after) and save it somewhere.
+### Requirements for usage
 
-### 2) Setup environment
+In order to have an account to comment as Feito in your PRs, a GitHub OAuth Token must be acquired. Follow this [link](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token). When the Scopes section arrives, the `admin:repo_hook` and `repo` checkboxes must be selected. Copy the Token (really, copy it, you won't be able visualize after) and save it somewhere.
```

Comment:
I think "you won't be able to visualize it afterward" would be better.

Reply:
Me too.